README.md

```diff
@@ -514,6 +514,18 @@ In place of common selectors like class, id or attribute we can use `document.qu
   target.parentNode.insertBefore(newEl, target.nextSibling);
   ```
 
+- [3.8](#3.8) <a name='3.8'></a> is
+
+  Returns `true` if it matches the query selector
+
+  ```js
+  // jQuery
+  $(e.target).is('.model-overlay');
+
+  // Native
+  el.matches('.model-overlay')
```

Comment:
`.model-overlay` is too specific, better to use `selector`.
`e.target` is too, better to use `$el` consistantly